docs/_examples/dkm_palettization.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
These should go into a separate PR on the `gh_pages` branch since these are generated files.

Reply:
I'm not sure if I need to do anything other than the usual routine of generating the html and copying that into `gh-pages`. Is there something more specific to do with tutorials? I didn't generate the html when you added the tutorials. I am also waiting for this PR to merge in order to generate the html. Thanks.

Reply:
The `_examples` folder should be a PR onto the `gh_pages` branch, _not_ the `main_` branch?

Reply:
@u-simha : Sorry, I don't know how to handle the `_examples` folder. It should show up in the `gh-pages` PR, which I am preparing now. My usual procedure is to wait for THIS PR to be merged before doing that. I made the only change that this PR covers, so this should be merged.

Reply:
Please remove it from this PR:

`git rm docs/_examples`
Commit & push

Reply:
@u-simha : There is no `_examples` folder in docs, so the above command shows `pathspec 'docs/_examples' did not match any files`.

Reply:
@u-simha : I think what happened was, I did a local preview (as I always do) of my PR change, before creating this PR. I did not know that the act of creating a preview introduced 18 file changes for a single-sentence deletion change in one file. That must be the `_examples` folder. And somehow it disappeared from my local clone, in my editing branch, because it is not there now. I will close this PR and start a new one, this time without any preview. Thanks!

Reply:
@u-simha : Done. Please approve this to unblock it. Thanks!